go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
 	github.com/openshift/api v0.0.0-20200205133042-34f0ec8dab87
+	github.com/redhat-cop/operator-utils v0.0.0-20190827162636-51e6b0c32776
```

Comment:
is it version from 2019?
Maybe better to depends on released v1.1.2 (Jan 30) https://github.com/redhat-cop/operator-utils/releases/tag/v1.1.2 ?

Reply:
Yeah, it seems it causes dependency issues, the furthest i can go is v0.1.0 so i've updated to that version - is this okay?

Reply:
that's OK, I just wondered if we can use the one that corresponds to our operator-sdk.

Reply:
The issue is controller-runtime -- we use `0.6.3`, the latest coputil uses `0.7.0`, and the latest version is `0.8.3`. I've never understood go dependency management well enough to figure out these issues though.

Reply:
`0.7.0` features quite a few [breaking changes](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.7.0), also